pages/glossary/background-colors-of-text.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Background Colors Of Text Nodes
+key: background-colors-of-text
+---
+
+All colors of the pixels not part of the [foreground colors](#foreground-colors-of-text) pixels, in an bounding box around each [visible](#visible) character in a [text node](https://dom.spec.whatwg.org/#text), where the width is the [advance width](https://www.w3.org/TR/css-values/#length-advance-measure
```

Comment:
The sentence formation where it goes - part of the foreground colors pixels, .... does not read right in my opinion.